### PR TITLE
Set up React application structure for admin 

### PR DESCRIPTION
**Pull request by DefyCab**
**THIS PR CONTAINS**

-scaffolds react app
-clean up and remove unnecessary files

@giacoletti @elvitak @Mljungho 

https://www.pivotaltracker.com/story/show/180923574

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,70 @@
-# yesterdays_news_admin
\ No newline at end of file
+# Getting Started with Create React App
```
Comment (giacoletti) [inline README.md]: You can set a title in the readme file like: "Yesterdays News (Admin)"

File: README.md [hunk 2]
```diff
@@ -1 +1,70 @@
...
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+
+### `yarn eject`
```
Comment (giacoletti) [inline README.md]: We agreed to remove this script, right?

File: README.md [hunk 3]
```diff
@@ -1 +1,70 @@
...
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+## Learn More
```
Comment (giacoletti) [inline README.md]: Since you're at it, you can delete all those lines in the readme file, from 42 till the end

File: package.json [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "5.0.0",
+    "web-vitals": "^2.1.0"
+  },
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test",
+    "eject": "react-scripts eject"
```
Comment (giacoletti) [inline package.json]: As we agreed, eject script can be removed

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
+    />
+    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
```
Comment (giacoletti) [inline public/index.html]: This reference can be deleted, along with the comments in this file

**Review by giacoletti — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (DefyCab) [reply README.md]: Roger!

File: README.md (hunk 2 above)
Comment (DefyCab) [reply README.md]: It wasn't in the chore, but we can take it away now. Handleing the script is in another chore.

File: public/index.html (hunk 5 above)
Comment (DefyCab) [reply public/index.html]: Affarmative!

File: package.json (hunk 4 above)
Comment (DefyCab) [reply package.json]: sure!

**Pull request merged**